lib/Conversion/TritonGPUToLLVM/Utility.cpp

```diff
@@ -413,8 +413,14 @@ void storeDistributedToShared(MemDescType dstTy, RankedTensorType srcTy,
                               const TargetInfoBase &target, bool crossGrain) {
   bool success;
   std::function<void(VectorType, Value /*shmemAddr*/)> perVectorCallback;
+  // Initialize variables in a safe scope. These variables will be bound
+  // to the lambda function.
+  // TODO: Enforce that crossGrain=True is only for AMD
```

Comment:
I was able to pass testing without determining the best way to do this, but in general this is a nice to have.